numba_cuda/numba/cuda/testing.py

```diff
@@ -189,6 +189,10 @@ def skip_if_cudadevrt_missing(fn):
     return unittest.skipIf(cudadevrt_missing(), "cudadevrt missing")(fn)
 
 
+def skip_if_nvjitlink_missing(reason):
```

Comment:
Not to be changed for this PR, but there are a few other test files that do something like this that should probably be changed to use this version of it.

Reply:
Yes, for example: 
https://github.com/NVIDIA/numba-cuda/blob/489045fe3e4b9d6de8d42bbde03e5a23dc416776/numba_cuda/numba/cuda/tests/cudadrv/test_nvjitlink.py#L44-L49